packages/ios_platform_images/ios/Classes/IosPlatformImagesPlugin.m

```diff
@@ -40,6 +40,99 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
       NSURL *url = [[NSBundle mainBundle] URLForResource:name withExtension:extension];
       result(url.absoluteString);
       return;
+    } else if ([@"loadSystemImage" isEqualToString:call.method]) {
+      if (@available(iOS 13, *)) {
+        NSString *name = call.arguments[0];
+        NSNumber *pixelSizeWithDouble = call.arguments[1];
```

Comment:
Is there a reason this isn't just:
```
double pixelSize = [(NSNumber*)(call.arguments[1]) doubleValue];
```

? It seems odd to keep the intermediate value.